packages/app/server/routes/publish.post.ts

```diff
@@ -319,6 +319,26 @@ export default eventHandler(async (event) => {
   return {
     ok: true,
     urls,
+    debug: {
+      workflowData: {
+        owner: workflowData.owner,
+        repo: workflowData.repo,
+        sha: workflowData.sha,
+        ref: workflowData.ref,
+      },
+      prAnalysis: {
```

Comment:
instead of new names, let's just put the actual variable names as object keys. so it'd be familiar to us.

```
      prAnalysis: {
        isPullRequest: isPullRequest(workflowData.ref),
```

it'd be 

```
        isPullRequest: isPullRequest(workflowData.ref),
```

and let's not create new values, just directly bringing the variables into the response (`debug`)